pinot-tools/src/main/java/org/apache/pinot/tools/streams/MeetupRsvpStream.java

```diff
@@ -20,102 +20,104 @@
 
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.node.ObjectNode;
-import java.io.IOException;
-import java.net.URI;
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
 import java.nio.charset.StandardCharsets;
 import java.util.Properties;
-import javax.websocket.ClientEndpointConfig;
-import javax.websocket.Endpoint;
-import javax.websocket.EndpointConfig;
-import javax.websocket.MessageHandler;
-import javax.websocket.Session;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 import org.apache.pinot.spi.stream.StreamDataProducer;
 import org.apache.pinot.spi.stream.StreamDataProvider;
 import org.apache.pinot.spi.utils.JsonUtils;
 import org.apache.pinot.tools.utils.KafkaStarterUtils;
-import org.glassfish.tyrus.client.ClientManager;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class MeetupRsvpStream {
   private static final Logger LOGGER = LoggerFactory.getLogger(MeetupRsvpStream.class);
   private StreamDataProducer producer;
   private boolean keepPublishing = true;
-  private ClientManager client;
 
-  public MeetupRsvpStream()
+  private ExecutorService _service;
+  private File _dataFile;
+
+  public MeetupRsvpStream(File dataFile)
```

Comment:
I think getting a live stream was there for a reason. Perhaps we should evaluate if that is a good idea, and if so, still keep it, as opposed to having a dataFile. Once it is a data-file, it is the same as any other provider (not necessarily meetup).